_pytest/assertion/rewrite.py

```diff
@@ -35,6 +35,12 @@
 REWRITE_NEWLINES = sys.version_info[:2] != (2, 7) and sys.version_info < (3, 2)
 ASCII_IS_DEFAULT_ENCODING = sys.version_info[0] < 3
 
+if sys.version_info >= (3,5):
```

Comment:
`getargspec` does not work on `Call`, `signature` does not exist on all pythons versions. Do you want a try/except ? It will also make it difficult to find dead code once 3.4 is not supported (which will happend one day I suppose)